explore/scripts/get_llnl_members.py

```diff
@@ -15,25 +15,25 @@
 authhead = helpers.get_gitauth()
 
 # Iterate through orgs of interest
-print "Gathering data across multiple paginated queries..."
+print("Gathering data across multiple paginated queries...")
 collective = {u'data': {}}
 tab = "    "
 
 for org in orglist:
 	pageNum = 1
-	print "\n'"+org+"'"
-	print tab+"page "+str(pageNum)
+	print("\n'"+org+"'")
```

Comment:
This might be cleaner as:

```python
print("\n%s'" % (org))
```

Which does this as a python string substitution.